src/lib/posenetWrapper.js

```diff
@@ -1,63 +1,101 @@
-import Bodypart from './bodypart';
-import MathModels from './mathModels'
+import Bodypart from "./bodypart";
+import MathModels from "./mathModels";
 
-class Pose{
+class Pose {
   constructor(posenetObject) {
     this.source = posenetObject;
     this._mm = new MathModels();
     this._bodypartIndexLookup = {
-      "nose" : 0, "leftEye" : 1, "rightEye" : 2,
-      "leftEar" : 3, "rightEar" : 4, "leftShoulder" : 5,
-      "rightShoulder" : 6, "leftElbow" : 7, "rightElbow" : 8,
-      "leftWrist" : 9, "rightWrist" : 10, "leftHip" : 11,
-      "rightHip" : 12, "leftKnee" : 13, "rightKnee" : 14,
-      "leftAnkle" : 15, "rightAnkle" : 16
+      nose: 0,
```

Comment:
this is gross. grosser than it already was

Reply:
how to resolve this?